test_conformance/subgroups/test_subgroup_ballot.cpp

```diff
@@ -926,11 +926,15 @@ template <typename T> int run_non_uniform_broadcast_for_type(RunTestForType rft)
 int test_subgroup_functions_ballot(cl_device_id device, cl_context context,
                                    cl_command_queue queue, int num_elements)
 {
-    std::vector<std::string> required_extensions = { "cl_khr_subgroup_ballot" };
```

Comment:
This variable is used in subhelpers.h removing makes dead code in that file.

Reply:
Yes, might as well remove it all. Done.